### PR TITLE
eos: Added new states eos_lacp

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

##### SUMMARY

Add new states gathered , rendered and parsed to eos_lacp.

related to #23

### CODE REVIEW

**Comment by GomathiselviS** [comment]
recheck

File: plugins/modules/eos_lacp.py [hunk 1]
```diff
@@ -37,8 +37,7 @@
 }
 
 DOCUMENTATION = """module: eos_lacp
-short_description: Manage Global Link Aggregation Control Protocol (LACP) on Arista
-  EOS devices.
+short_description: Lacp resource module.
```
Comment (rohitthakur2590) [inline plugins/modules/eos_lacp.py:40]: IMHO  LACP resource module (but you can have these changes  with script in different PR )

File: plugins/module_utils/network/eos/argspec/lacp/lacp.py [hunk 2]
```diff
@@ -47,8 +47,17 @@ def __init__(self, **kwargs):
             },
             "type": "dict",
         },
+        "running_config": {"type": "str"},
         "state": {
-            "choices": ["merged", "replaced", "deleted"],
+            "choices": [
+                "merged",
+                "replaced",
+                "overridden",
```
Comment (rohitthakur2590) [inline plugins/module_utils/network/eos/argspec/lacp/lacp.py:55]: I think Overridden not supported here.

File: plugins/module_utils/network/eos/config/lacp/lacp.py [hunk 3]
```diff
@@ -99,9 +120,18 @@ def set_state(self, want, have):
                   to the desired configuration
         """
         state = self._module.params["state"]
+        if (
+            state in ("merged", "replaced", "overridden", "rendered")
```
Comment (rohitthakur2590) [inline plugins/module_utils/network/eos/config/lacp/lacp.py:124]: If not supported then we need to remove overridden.

File: plugins/modules/eos_lacp.py [hunk 4]
```diff
@@ -170,8 +226,19 @@ def main():
 
     :returns: the result form module invocation
     """
+    required_if = [
+        ("state", "merged", ("config",)),
+        ("state", "replaced", ("config",)),
+        ("state", "overridden", ("config",)),
```
Comment (rohitthakur2590) [inline plugins/modules/eos_lacp.py:232]: overridden not required.

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
gate

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Pull request merged**